README.md

```diff
@@ -3,6 +3,8 @@ This package was originally authored by [Allardvm](https://github.com/Allardvm)
 LightGBM.jl
 ![CI](https://github.com/IQVIA-ML/LightGBM.jl/workflows/CI/badge.svg)
 [![License](http://img.shields.io/badge/license-MIT-brightgreen.svg?style=flat)](LICENSE.md)
+[![Stable](https://img.shields.io/badge/docs-stable-blue.svg)](https://IQVIA-ML.github.io/LightGBM.jl/stable)
+[![Dev](https://img.shields.io/badge/docs-dev-blue.svg)](https://IQVIA-ML.github.io/LightGBM.jl/dev)
```

Comment:
Whats the difference between dev and stable?

Reply:
The development version is the document that created the document from the latest committed code.
A stable version is a document created from the latest release of code.

Reply:
Sorry I mean to clarify -- how will the system know whats dev and whats stable? Will we have to do something periodically manually to make the right things appear at dev/stable?

Reply:
The system automatically determines if the document is stable based on the release tag. So you don't have to do anything manually on a regular basis.